doc/contributing.rst

```diff
@@ -211,12 +211,12 @@ Advanced tips and tricks
 
 Here are some advanced tips and tricks when working with RSMTool tests.
 
-#. To run a specific test function in a specific test file, simply use ``nosetests --nologcapture tests/test_X.py:Y`` where ``test_X.py`` is the name of the test file, and ``Y`` is the test functions. Note that this will not work for parameterized tests. If you want to run a specific parameterized test, you can comment out all of the other ``param()`` calls and run the ``test_run_experiment_parameterized()`` function as above.
+#. To run a specific test function in a specific test file, simply use ``nose2 --quiet tests test_X.Y.Z`` where ``test_X`` is the name of the test file, and ``Y`` is the ``unittest.TestCase`` subclass. ``Z`` is the test function. Note that this will not work for parameterized tests. If you want to run a specific parameterized test, you can comment out all of the other parameters in the ``params``` and run the ``test_run_experiment_parameterized()`` function as above.
```

Comment:
```suggestion
#. To run a specific test function in a specific test file, simply use ``nose2 --quiet tests test_X.Y.Z`` where ``test_X`` is the name of the test file, ``Y`` is the enclosing ``unittest.TestCase`` subclass, and ``Z`` is the desired test function. Note that this will not work for parameterized tests. If you want to run a specific parameterized test, you can comment out all of the other parameters in the ``params``` and run the ``test_run_experiment_parameterized()`` function as above.
```